metrics/metricstest/metrics_mock.go

```diff
@@ -0,0 +1,132 @@
+package metricstest
+
+import (
+	"github.com/prebid/prebid-cache/config"
+	"github.com/prebid/prebid-cache/metrics"
+	"time"
+)
+
+var MockHistograms map[string]float64
+var MockCounters map[string]int64
+
+func CreateMockMetrics() *metrics.Metrics {
+	MockHistograms = make(map[string]float64, 6)
+	MockHistograms["puts.current_url.duration"] = 0.00
+	MockHistograms["gets.current_url.duration"] = 0.00
+	MockHistograms["puts.backends.request_duration"] = 0.00
+	MockHistograms["puts.backends.request_size_bytes"] = 0.00
+	MockHistograms["gets.backends.duration"] = 0.00
+	MockHistograms["connections.connections_opened"] = 0.00
+	MockHistograms["extra_ttl_seconds"] = 0.00
+
+	MockCounters = make(map[string]int64, 16)
+	MockCounters["puts.current_url.request.total"] = 0
+	MockCounters["puts.current_url.request.error"] = 0
+	MockCounters["puts.current_url.request.bad_request"] = 0
+	MockCounters["gets.current_url.request.total"] = 0
+	MockCounters["gets.current_url.request.error"] = 0
+	MockCounters["gets.current_url.request.bad_request"] = 0
+	MockCounters["puts.backends.add"] = 0
+	MockCounters["puts.backends.json"] = 0
+	MockCounters["puts.backends.xml"] = 0
+	MockCounters["puts.backends.invalid_format"] = 0
+	MockCounters["puts.backends.defines_ttl"] = 0
+	MockCounters["puts.backends.request.error"] = 0
+	MockCounters["puts.backends.request.bad_request"] = 0
+	MockCounters["gets.backends.request.total"] = 0
+	MockCounters["gets.backends.request.error"] = 0
+	MockCounters["gets.backends.request.bad_request"] = 0
+	MockCounters["connections.connection_error.accept"] = 0
+	MockCounters["connections.connection_error.close"] = 0
+
+	return &metrics.Metrics{
+		MetricEngines: []metrics.CacheMetrics{
+			&MockMetrics{
+				MetricsName: "Mockmetrics",
+			},
+		},
+	}
+}
+
+type MockMetrics struct {
+	MetricsName string
+}
+
+func (m *MockMetrics) Export(cfg config.Metrics) {
```

Comment:
Super Nitpick: Consider adding a line break between each method for readability.

Reply:
This is not exactly in practice throughout the repo so a PR devoted to adding this spaces to the functions in all files would be better. Edited the PR to add a line break in most code I touched though.